app/posts/apply-for-teacher-training/2019-12-18-digital-accessibility-centre-report.md

```diff
@@ -0,0 +1,110 @@
+
+### Non-descriptive form elements (WCAG AA)
+
+Page 27: A button to manage which candidates are used for reporting did not have an adequate description.
+
+> The ‘Hide in reporting’ and ‘Show in reporting’ buttons are not descriptive enough for all users, particularly screen reader users, to determine their function or purpose.
+
+This was [fixed by expanding the description of the button](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1025). ([Trello](https://trello.com/c/LIxkhgmi/696-dac-page-27-add-context-to-show-hide-in-reporting-button-in-support-application-page))
+
+Automated accessibility testing would not have found this issue.
+
+### Status message (WCAG AA)
+
+Page 29: When a success message was shown following an action the message was not focused, meaning screenreader users would not be aware of it.
+
+This was fixed [by making it focusable](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/963) and [by adding an aria alert role to the element](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/969). ([Trello](https://trello.com/c/83AycElp/683-dac-fix-success-message-focus))
+
+The fix brings our success alert inline with the error summary component provided by the design system.
+
+Automated accessibility testing would not have found this issue.
+
+## WCAG AAA issues
+
+We also fixed the following issues:
+
+* [Page 33: Add hidden content to "Change" links on Work History review (and other) page(s)](https://trello.com/c/XsqlA0hu/698-dac-page-33-add-hidden-content-to-change-links-on-work-history-review-and-other-pages)
+* [Page 41: Change headings on provider offer page so they are in a hierarchical order](https://trello.com/c/n0wiBbfB/699-dac-page-41-change-headings-on-provider-offer-page-so-they-are-in-a-hierarchical-order)
+* [Page 50: Add (optional) to standard conditions in provider applications offer page](https://trello.com/c/hW5m9EFg/702-dac-page-50-add-optional-to-standard-conditions-in-provider-applications-offer-page)
+* [Page 51: Change all pages to only ask for one thing per page](https://trello.com/c/QJVmXusT/703-dac-page-51-change-all-pages-to-only-ask-for-one-thing-per-page)
+* [Page 56: Add error to the page title when rejecting an application without passing validation](https://trello.com/c/RK2v971U/704-dac-page-56-add-error-to-the-page-title-when-rejecting-an-application-without-passing-validation)
+* [Page 59: Add custom title to "Respond to application" provider page](https://trello.com/c/ARnK0YPP/706-dac-page-59-add-custom-title-to-respond-to-application-provider-page)
+* [Page 60: Add custom title to "Confirm offer" provider page](https://trello.com/c/Q7H4sXsR/707-dac-page-60-add-custom-title-to-confirm-offer-provider-page)
+* [Page 61: Add type="number" to "Graduation year" field in candidate degrees page](https://trello.com/c/QKnJCS65/708-dac-page-61-add-typenumber-to-graduation-year-field-in-candidate-degrees-page)
```

Comment:
This probs needs changing to inputmode as per the impending update from GOV.UK Frontend.

Reply:
Make a ticket?